tests/model_registry/model_registry/gateway/test_gateway_domain.py

```
@@ -0,0 +1,156 @@
+
+    @pytest.mark.sanity
+    def test_model_registry_reachable_via_gateway_domain(
+        self,
+        gateway_model_registry_url: str,
+    ):
+        """Given GATEWAY_DOMAIN is set and HTTPRoutes exist for model registry
+        When sending a request to the gateway domain with the model registry path
+        Then the gateway routes the request (401 Unauthorized is acceptable)
+        """
+        LOGGER.info(f"Testing gateway connectivity: {gateway_model_registry_url}")
+        response = requests.get(gateway_model_registry_url, verify=False, timeout=30)
```

Ruff (0.15.13)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)